### ISSUE TITLE
Docker build fails on 0.4.7 and 0.5.0

### ISSUE THREAD

**Issue by plakhin** [laravel-shift/workbench-issues#34]
**Desktop:**
 - OS: Windows
 - OS Version: 10
 - App Version: 0.5.0

**Describe the bug**
Docker build fails after update to 0.4.7 and 0.5.0, though worked fine on 0.4.6:
```
#8 [ 5/14] RUN adduser -D -u -1 shift
#8 sha256:fd91e786836ea8626df9570a8ba8c9ef87fa309894547ddadd58520276225572
#8 0.490 adduser: invalid number '-1'
#8 ERROR: executor failed running [/bin/sh -c adduser -D -u -1 shift]: exit code: 1
------
 > [ 5/14] RUN adduser -D -u -1 shift:
------
executor failed running [/bin/sh -c adduser -D -u -1 shift]: exit code: 1
```

**To Reproduce**
1. Update to 0.4.7 or 0.5.0
2, Try to run any task

![image](https://user-images.githubusercontent.com/25561058/128580248-429e921b-0e89-439a-a759-2387d2c7e7e4.png)

**Comment by jasonmccreary** [comment]
We added this to solve an issue on Linux, but looks like it now affects Windows. Are you using WSL?

**Comment by jessarcher** [comment]
If PR'd a fix for this so will hopefully be able to release it soon :+1:

**Comment by plakhin** [comment]
> We added this to solve an issue on Linux, but looks like it now affects Windows. Are you using WSL?

Yes, WSL2

**Comment by jasonmccreary** [comment]
This should be resolved in the latest release (0.5.1). Let us know if not.

**Comment by plakhin** [comment]
0.5.1 works fine, thanks!
